lib/1.4/dml-builtins.dml

```diff
@@ -551,8 +551,10 @@ template device {
     param be_bitorder             default _be_bitorder;
     param log_group                 = undefined;
 
-    param use_io_memory             default simics_api_version == "6"
-        || simics_api_version == "5" || simics_api_version == "4.8";
+    param use_io_memory             default simics_api_version == "7"
+        || simics_api_version == "6"
+        || simics_api_version == "5"
+        || simics_api_version == "4.8";
```

Comment:
io_memory is a legacy feature that we want to deprecate, so please don't extend "default true" into the 7 API

Reply:
Ok you're saying I should instead set all the devices using it to SIMICS_API = 6.

Reply:
... Or that devices with banks using it should set `param use_io_memory = true` explicitly. Note that top-level `use_io_memory` is just a default for banks within the device, and banks can override it locally on a bank-by-bank basis.

Reply:
Or alternatively, explicitly set `use_io_memory = true` in those devices.

Reply:
And not just in devices *using* it, but in devices actually *needing* it. Most "normal" devices should work fine wtih a silent switch to the transaction iface.

Reply:
I created SIMICS-20984 for this.